cip-constitution-tech/README.md

```diff
@@ -0,0 +1,261 @@
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/796
+Created: 2024-03-19
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Cardano's minimum viable governance model as described within [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md) introduces the concept of a Cardano constitution.
+Although CIP-1694 gives no definition to the constitution's content or form.
+
+This proposal aims to describe a standardized technical form for the Cardano Constitution to enhance the accessibility and safety of the document.
+
+> **Note:** This proposal only covers the technical form of the constitution, this standard is agnostic to the content of the constitution.
+
+## Motivation: why is this CIP necessary?
+
+CIP-1694 defines the on-chain anchor mechanism used to link the off-chain Constitution document to on-chain actions.
+This mechanism was chosen due to its simplicity and cost effectiveness, moving the potentially large Cardano constitution off-chain, leaving only a hash digest and URI on-chain.
+This is the extent to which CIP-1694 outlines the Cardano Constitution: CIP-1694 does not provide suggestions around hashing algorithm, off-chain storage best practices or use of rich text styling.
+
+By formalizing the form of the constitution and its iterations, we aim to promote its longevity and accessibility.
+This is essential to ensure the effectiveness of the CIP-1694 governance model.
+
+This standard will impact how Ada holders read the Constitution but the main stakeholders for this are the tool makers who wish to read, render and write a constitution.
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution, we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform, we can ensure its immutability and permissionless access.
+This is a step to improve the longevity and accessibility of each constitution iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make the role of constitution iteration comparison tools easier, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification SHOULD be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to replace this one.
+
+### Terminology
+
```

Comment:
> If you want I will prepare that as a code diff.

@rphair
I do think these rules are true and helpful
Please add these rules into this section 💪

Reply:
@Ryun1 added under `### Terminology`:

---

#### Capitalisation of key terms

To avoid unnecessary edits, and therefore checksum changes, constitution authors MUST follow the following standard English capitalisation rules unless a translation language indicates otherwise:

`Constitution` vs. `constitution`

- The Constitution in effect — either the "initial" one or any updated version — is unique and therefore capitalised ("Constitution") as a proper noun.
- Draft or proposed constitutions are not unique & therefore are not capitalised ("constitution") as a common noun.
- "Cardano Constitution" is a very specific proper noun phrase (also a title) and so each word is capitalised.
- The phrase "the Constitution", unless used non-specifically (e.g. "the constitution that voters prefer"), would generally be assumed to be _the_ Constitution and capitalised as a proper noun.

`Ada` vs. `ada` vs. `ADA`

- `Ada` is the currency, while `ada` indicates units of that currency (e.g. "Ada holders can accumulate more ada to increase their influence.")
- `ADA` is the trading symbol (e.g. "Fluctuations in ADA might influence decisions about the Treasury.")

Reply:
honestly a Cardano technical terminology glossary, would be a useful thing
has this been done before?

Reply:
@Ryun1 the closest thing I know of that type is this page on the Dev Portal (maybe @katomm knows of another?): https://developers.cardano.org/docs/portal-style-guide/#editorial-style-guide